reo/src/emissions_calculator.py

```diff
@@ -24,120 +24,154 @@ def __init__(self, latitude=None, longitude=None, time_steps_per_hour=1, **kwarg
         self.library_path = os.path.join('reo', 'src', 'data')
         self.latitude = float(latitude) if latitude is not None else None
         self.longitude = float(longitude) if longitude is not None else None
-        self.time_steps_per_hour = time_steps_per_hour
         self._region_abbr = None
         self._emmissions_profile = None
         self._transmission_and_distribution_losses = None
         self.meters_to_region = None
-        
+        self.time_steps_per_hour = kwargs.get('time_steps_per_hour') or 1
         proj102008 = pyproj.Proj("+proj=aea +lat_1=20 +lat_2=60 +lat_0=40 +lon_0=-96 +x_0=0 +y_0=0 +datum=NAD83 +units=m +no_defs")
         self.project4326_to_102008 = partial(pyproj.transform,pyproj.Proj(init='epsg:4326'),proj102008)
     
     @staticmethod
     def add_to_data(data):
 
-        if 'Emissions Warning' in (data['messages'].get('warnings') or {}).keys():
```

Comment:
@t-kwasnik should this if block in emissions_calculator be removed?

Reply:
@NLaws yes, it is not needed